packages/itwin/reports-config-widget/src/widget/components/ExtractionStatus.scss

```diff
@@ -0,0 +1,29 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+@import "~@itwin/itwinui-css/scss/variables";
+
+.iui-message {
```

Comment:
Do not override global styles. What if another widgets in the same applications use the `iui-message` class?

Reply:
`.iui-message` style override has been deleted